xapian-bindings/python3/docs/introduction.rst

```diff
@@ -28,6 +35,11 @@ There's also
 `simplematchdecider.py <examples/simplematchdecider.py>`_
 which shows how to define a MatchDecider in Python.
 
+Examples specific to letor are:
+`xapian-prepare-trainingfile.py <examples/xapian-prepare-trainingfile.py>`_.,
+`xapian-train.py <examples/xapian-train.py>`_.,
```

Comment:
The two commas above both have an unwanted full stop before them - in the rendered output this looks like:

```
Examples specific to letor are: xapian-prepare-trainingfile.py., xapian-train.py., xapian-rank.py.
```

Reply:
fixed in  a fixup commit.